addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -498,6 +498,235 @@ registry.slider = publicWidget.Widget.extend({
         this._computeHeights();
     },
 });
+registry.backgroundVideo = Animation.extend({
+    selector : ".oe_video_background",
+    disabledInEditableMode : false,
+
+    start: function () {
```

Comment:
Add the documentation tags, regroups the private / public function, use camelCase variable and function, etc :wink: